identity-service/src/config.js

```diff
@@ -843,7 +843,7 @@ const config = convict({
     doc: 'Enable replica set updates with Entity Manager',
     format: Boolean,
     env: 'entityManagerReplicaSetEnabled',
```

Comment:
we can get rid of this altogether too

Reply:
good point - i was just tryna limit scope of this change but maybe i'll just rip it all out rn